config.sample.yaml

```diff
@@ -0,0 +1,92 @@
+---
+# After you have replaced all the __CHANGE_ME__ values in this file, you need to
+# run `make auth` to setup the Auth.
+management:
+  location: __CHANGE_ME__
```

Comment:
should be common - top level

Reply:
are you saying that all of these values should be moved to the top level, or just the location?

If it's just the location, that probably makes sense.
If it's the whole section, I'm not sure I agree.

Reply:
I've updated the file structure, so the tre_id and the location are at top level and and everything else under "tre"